allennlp/modules/text_field_embedders/basic_text_field_embedder.py

```diff
@@ -96,6 +98,9 @@ def forward(self, text_field_input: Dict[str, torch.Tensor], num_wrapping_dims:
             # Note: need to use getattr here so that the pytorch voodoo
             # with submodules works with multiple GPUs.
             embedder = getattr(self, 'token_embedder_{}'.format(key))
+            identifiers = {}
+            if isinstance(embedder, MultilangTokenEmbedder):
```

Comment:
We really need to use `inspect` here, instead of hard-coding a particular embedder.  It's not worth the complexity to support one-off embedders in the core components of the main library like this, unless we can find a way to do it generally.  If you don't know how to do this, I can give some pointers.